peeratmac/rancid-tomatillos
# Util

### What's this PR do?
- Begins linting by removing unnecessary imports and correcting lines of 80 characters
- Refactors `findUserRatings` and `findRatingId` methods in util.js
- Tests findRating function in util.js

### Where should the reviewer start?
- Please look at the refactoring completed on MovieCard.js and MovieShowPage (and the associated MovieShowPage tests).  The test suite is not yet completed due to issues with testing `updateUser` prop.

### How should this be manually tested?
- Run test suite to see that all are passing.
- The tests that I have had issues with are commented out with notes about why they are failing.

### What are the relevant tickets?
Issue #26 and Issue #22
